indico/modules/events/reminders/templates/preview_html.html

```diff
@@ -0,0 +1,4 @@
+<strong>{% trans %}Subject{% endtrans %}</strong>
+<br>{{ subject }}<br><br>
+<strong>{% trans %}Body{% endtrans %}</strong>
+{{ body | safe }}
```

Comment:
That sounds like a great way to self-XSS. You most likely want the `sanitize_html` filter instead.